### ISSUE TITLE
User admin webinterface

### ISSUE THREAD

**Issue by boyska** [insomnia-lab/libreant#78]
We should be able to do basic user operations through a web interface. This is simlar to #77, but is about a web interface instead of a CLI one.

This depends on #6

**Comment by ael-code** [comment]
I'd like to use a simple template.
I found this nice project https://github.com/IronSummitMedia/startbootstrap-sb-admin-2

**Comment by boyska** [comment]
ael wrote:
> I found this nice project
> https://github.com/IronSummitMedia/startbootstrap-sb-admin-2

very puccettose, so many colors, great choice.
go on!

**Comment by ael-code** [comment]
Can we postpone this to the next release?

**Comment by boyska** [comment]
yes, moving to 0.4
